.eslintrc

```diff
@@ -31,6 +37,8 @@
     "no-undef": 2,
     "no-use-before-define": 2,
     "brace-style": [1,"1tbs",{}],
-    "no-mixed-spaces-and-tabs": 2
+    "no-mixed-spaces-and-tabs": 2,
+    "quotes": ["error", "single"],
```

Comment:
We may have a style fight here. I prefer single quotes. Never mind, I was interpreting this incorrectly. I like single spaces 😄